task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="logo.png">
+    <title> Ribbitter | Feed </title>
+</head>
+<body>
+<header>
+    <nav class="container">
+        <a class="logo" href="index.html">
+            <img src="logo.png" alt="A VERY MUCH GREEN FROG">
+            <span class="logotext"><h1>Ribbitter</h1></span>
+        </a>
+        <a class="write" href="">
+            <img src="edit.png" alt="Write new post">
+        </a>
+        <a class="exit" href="">
+            <img src="exit.png" alt="exit">
+        </a>
+        <a class="user" href="">
+            <span class="yourname">YourName</span>
+            <img src="you.png" alt="your photo">
+        </a>
+    </nav>
+</header>
+<aside>
+    <h2 class="prefs">Your content preferences:</h2>
+    <form id="filterform">
+    <div class="filter">
+        <input type="checkbox"><span>Sort by name</span>
+        <form action="" method="get" id="searchname">
+            <input type="text" placeholder="Type name..." class="typein">
+        </form></input>
+        <form><p></p></form>
+        <input type="checkbox"><span>Sort by date</span>
+        <form action="" method="get" id="searchdate">
+            <input type="text" placeholder="dd" class="day">
+            <input type="text" placeholder="mm" class="month">
+            <input type="text" placeholder="yyyy" class="year">
+        </form></input>
+        <form><p></p></form>
+        <input type="checkbox"><span>Sort by tags</span>
+        <form action="" method="get" id="searchtags">
+            <input type="text" placeholder="Type tag..." class="typein">
+        </form></input>
+        <form><p></p></form>
+    </div>
+    </form>
+    <button class="filtersubmit" form="filterform">Submit</button>
+</aside>
+<main>
+    <article class="post">
+        <div class="userpost">
+            <img class="postphoto" src="ofpic.png" alt="Photo of user">
+            <a class="username" href="">RibbitterOficial</a>
+        </div>
+
+        <div class="datepost">
+            <br>18.02.2020</br>
+            <br>15:15</br>
+        </div>
+        <div class="text">
+            <br>
+            I went to the woods because I wished to live deliberately, to front only the essential facts of life, and see if I could not learn what it had to teach, and not, when I came to die, discover that I had not lived.
+            </br>
+            <br>(c) Henry David Thoreau </br>
+        </div>
+        <div class="tagspost">
+            <span class="showtags" onclick="">Show tags</span>
```

Comment:
onclick is not needed here
if the element should bring the action ti should be actionable by itself, button in that situation